source/_components/media_player.volumio.markdown

```diff
@@ -0,0 +1,32 @@
+---
+layout: page
+title: "Volumio Media Player"
+description: "How to set up the Volumio media player platform"
+date: 2017-03-04
+sidebar: true
+comments: false
+sharing: true
+footer: true
+ha_category: Media Player
+ha_release: 0.40
+---
+
+The volumio platform allows you to control a [Volumio](http://volumio.org) media player
+from Home Assistant.
+
+
+To add a Volumio player to your installation, add the following to
+your configuration.yaml file.
+
+# Example configuration.yaml entry
```

Comment:
Formatting of example is broken. Have a look at the [PI GPIO](https://home-assistant.io/components/switch.rpi_gpio/) docs as en example.